src/dawn/IIR/AccessToNameMapper.h

```diff
@@ -15,7 +15,8 @@
 #ifndef DAWN_IIR_ACCESSTONAMEMAPPER_H
 #define DAWN_IIR_ACCESSTONAMEMAPPER_H
 
-#include "dawn/SIR/ASTVisitor.h"
+#include "dawn/IIR/ASTFwd.h"
```

Comment:
why do we now need both includes?

Reply:
IIR/ASTFwd.h to get forward declarations of Stmt, BlockStmt, Expr, ... while IIR/ASTVisitor.h to get the full declaration of the base ASTVisitor.